server/docker/docker-compose.yml

```diff
@@ -1,27 +1,28 @@
-version: "3"
+version: "4"
 # OPTIMIZE: use docker compose instead of `run-locally.sh`
 
-#services:
-#  vocitrainer-dev:
-#    build:
-#      context: $PWD/../
-#      dockerfile: $PWD/Dockerfile
-#    ports:
-#      - "51337:51337"
-#    environment:
-#      PYTHONPATH: ./src
-#      analytics_api_url: "http://localhost:10000"
-#      feedback_api_url: "https://dev-api.taskbase.com"
-#      api_key: "xxx"
-#      tenant_id: "99"
-#
-#    command:
-#      [
-#        "uvicorn",
-#        "main:app",
-#        "--host",
-#        "0.0.0.0",
-#        "--port",
-#        "51337",
-#        "--reload",
-#      ]
+services:
```

Comment:
since you removed the comments, does it work? If so, then we should get rid of `server/run-locally.sh` as that was only a fallback

Reply:
It worked for me - I can get rid of `run-locally.sh` then